source/MaterialXView/Handlers/GeometryHandler.h

```diff
@@ -3,12 +3,18 @@
 
 #include <string>
 #include <memory>
+#include <vector>
 
 namespace MaterialX
 {
 /// Shared pointer to an GeometryHandler
 using GeometryHandlerPtr = std::shared_ptr<class GeometryHandler>;
 
+/// Geomtry index buffer
+using GeometryIndexBuffer = std::vector<unsigned int>;
+/// A float geometry buffer
+using FloatGeometryBuffer = std::vector<float>;
```

Comment:
Maybe name these IndexBuffer and FloatBuffer, and put them inside GeometryHandler.